docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,312 @@
+
+1. `name` (REQUIRED). The name of the assessment.
+2. `date` (REQUIRED). The date at which the assessment is completed.
+3. `contents`. There can be multiple items in contents. For each item the following fields are present:
+
+    1. `question` (REQUIRED). A question.
+    2. `answer` (REQUIRED). An answer.
+    3. `remarks` (OPTIONAL). A field to put relevant discussion remarks in.
+    4. `authors`. There can be multiple names. For each name the following field is present.
+        1. `name` (OPTIONAL). The name of the author of the question.
+    5. `timestamp` (OPTIONAL). A timestamp of the date and time of the answer.
+
+### `model_card`
+
+A `model_card` contains the following information.
+
+1. `algorithm-registers` (OPTIONAL, list[string]). If this algorithm is registered in an algorithm register,
+this field should contain a URI to the corresponding record in the register. There can be multiple registers.
+A URI could look like `https://huggingface.co/org/modelid` or `https://algoritmes.overheid.nl/nl/algoritme/modelid`.
+2. `language` (OPTIONAL, list[string]). If relevant, the natural languages the model supports in [ISO 639](https://www.iso.org/iso-639-language-code).
+    There can be multiple languages.
+3. `license`(REQUIRED, string). Any license from the [open source license list](https://opensource.org/license)
+[^1]. If the license is NOT present in the license list this field must be set to 'other' and the following
+two fields will be REQUIRED.
+
+    1. `license_name` (string). An id for the license.
+    2. `license_link` (string). A link to a file of that name inside the repo, or a URL to a remote file containing the license
+    contents.
+
+4. `library_name` (OPTIONAL, string). Any library from the [library list](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts).
+    If the library is NOT present in the library list, the following fields CAN be set.
+    1. `library` (OPTIONAL, string). If a library is used that is not in the above list, you can provide
+    the name of the library used here.
+    2. `library_link` (OPTIONAL, string). If a libary is used that is not in the above list, you can
+    provide a URI to the repo of the library used here.
+5. `tags` (OPTIONAL, list[string]). Tags with keywords to describe the project. There can be multiple tags.
+
+#### `model_index`
+
+There can be multiple models. For each model the following fields are present.
+
+1. `name` (REQUIRED, string). The name of the model.
+2. `model` (REQUIRED, string). A URI pointing to a repository containing the model file.
+3. `artifacts` (OPTIONAL, list[string]). A list of URI's where each URI refers to a relevant model
+artifact, that cannot be captured by any other field, but are relevant to model.
+4. `parameters` (list). There can be multiple parameters. For each parameter the following fields are present.
```

Comment:
I would propose to also add `labels` here. F.e. a parameter can come from "a source" and  you would like to have a way to store this meta information. Labels can be a way to do this.

Reply:
Do you mean a labels field under the parameters field or a labels field on the same level als the parameters fields?